README.md

```diff
@@ -1,12 +1,40 @@
-# Repo for SRE Content
+# Operate First SRE Content
 
-##  Material
+## Introduction
 
-* [Architecture Decision Records](./ADRs/)
-* [SRE Maturity Model](./sre_maturity.md)
-* [Picking Good SLIs](./picking_good_slis.md)
+This the Operate First repository for Site Reliability documentation. Here we
+intend to collect documentation on the nitty-gritty aspects of building and
+operating an effective Site Reliability organisation. 
```

Comment:
let's stick to using the full "Site Reliability Engineering" please

Reply:
-> “SRE” (as above)

Reply:
Some folks e.g. Disney are calling it Systems Reliability Engineering, we may be fine with just one expansion of the acronym in the project as long as we are consistent with the full spell out on first usage plus the (SRE) parenthetical, then SRE thereafter.